destinationCreation.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+      <meta charset="UTF-8"/>
+      <title>Destination Creation</title>
+      <link rel="stylesheet" href="destinationCreation.css"/>
+      <script src="destinationCreation.js"></script>
+    </head>
+    <header>
+    </header>
+    <body>
+      <div id="content">
+        <h1>Destination Creation Form</h1>
+        <div id="form-container">
+          <form action="/data" method="post">
+            <label for="name-input">Enter the name of the destination: *</label>
+            <textarea name="name" id="name-input" required></textarea>
+            <p>Enter the latitude and longitude of the destination:</p>
+            <label for="lat-input">Latitude: *</label>
+            <textarea name="latitude" id="lat-input" required></textarea>
+            <label for="long-input">Longitude: *</label>
+            <textarea name="longitude" id="long-input" required></textarea>
+            <label for="city-input">Enter the city where the destination is located: *</label>
+            <textarea name="city" id="city-input" required></textarea>
+            <label for="riddle-input">Enter a riddle for this destination: *</label>
+            <textarea name="riddle" id="riddle-input" required></textarea>
+            <p>Enter 1-3 hints for this destination that provides clarity for the riddle:</p>
+            <label for="hint1-input">Hint 1: *</label>
+            <textarea name="hint1" id="hint1-input" required></textarea>
+            <label for="hint2-input">Hint 2:</label>
+            <textarea name="hint2" id="hint2-input"></textarea>
+            <label for="hint3-input">Hint 3:</label>
+            <textarea name="hint3" id="hint3-input"></textarea>
+            <label for="desc-input">Enter a description for this destination: *</label>
+            <textarea name="description" id="desc-input" required></textarea>
+            <p>Check all the tags that apply to this destination: *</p>
+            <input type="checkbox" name="food" id="food-input">
+            <label for="food-input">Food</label>
+            <input type="checkbox" name="sports" id="sports-input">
+            <label for="sports-input">Sports</label>
+            <input type="checkbox" name="historical" id="historical-input">
+            <label for="historical-input">Historical</label>
+            <input type="checkbox" name="art" id="art-input">
+            <label for="art-input">Art</label>
+            <input type="checkbox" name="tourist" id="tourist-input">
+            <label for="tourist-input">Tourist</label>
+            <input type="checkbox" name="family" id="family-input">
+            <label for="family-input">Family</label>
+            <button type="submit">Submit</button>
+          </form>
+        </div>
+        <p>* = a required field</p>
+      </div>
+    </body>
+</html>
```

Comment:
The HTML Presubmit requires an empty line at the end of html/css files.

Reply:
ok good to know. Thanks